spec/unit/commands/activedocs_command/apply_command_spec.rb

```diff
@@ -21,7 +22,7 @@
 
   context '#run' do
     context 'when --publish and --hide set' do
-      let(:options) { { publish: true, hide: true } }
+      let(:options) { default_options.merge({ publish: true, hide: true }) }
```

Comment:
Rubocop or any ruby linter does not complain for ` Redundant curly braces around a hash parameter`?

Reply:
I was not aware of the need of usage of a linter in this project. I've fixed this and enabled rubocop in my IDE :+1: